packages/client/src/hooks/useTreasury.js

```diff
@@ -1,35 +1,18 @@
 import { useEffect, useReducer } from "react";
-import { mutate, query, tx, config } from "@onflow/fcl";
-import { syncSafeOwnersWithSigners, getVaultId, removeAddressPrefix } from "../utils";
-import { COIN_TYPE_TO_META } from "constants/maps";
-import reducer, { INITIAL_STATE } from "../reducers/treasuries";
+import { mutate, query, tx } from "@onflow/fcl";
 import {
   CREATE_TREASURY_LIMIT,
   EXECUTE_ACTION_LIMIT,
   SIGNED_LIMIT,
   SIGNER_APPROVE_LIMIT,
-  UPDATE_SIGNER_LIMIT,
+  UPDATE_SIGNER_LIMIT
 } from "constants/constants";
-import { createSignature } from "../contexts/Web3";
-import {
-  GET_SIGNERS,
-  GET_THRESHOLD,
-  INITIALIZE_TREASURY,
-  GET_PROPOSED_ACTIONS,
-  SIGNER_APPROVE,
-  GET_SIGNERS_FOR_ACTION,
-  EXECUTE_ACTION,
-  GET_TREASURY_IDENTIFIERS,
-  UPDATE_THRESHOLD,
-  ADD_SIGNER,
-  UPDATE_SIGNER,
-  REMOVE_SIGNER,
-  GET_VAULT_BALANCE,
-  PROPOSE_TRANSFER,
-  ADD_VAULT,
-  ADD_COLLECTION,
-} from "../flow";
 import { COIN_TYPES } from "constants/enums";
+import { COIN_TYPE_TO_META } from "constants/maps";
+import { createSignature } from "contexts/Web3";
+import { ADD_SIGNER, EXECUTE_ACTION, GET_PROPOSED_ACTIONS, GET_SIGNERS, GET_SIGNERS_FOR_ACTION, GET_THRESHOLD, GET_TREASURY_IDENTIFIERS, GET_VAULT_BALANCE, INITIALIZE_TREASURY, PROPOSE_TRANSFER, REMOVE_SIGNER, SIGNER_APPROVE, UPDATE_SIGNER, UPDATE_THRESHOLD } from "../flow";
```

Comment:
Suggest prettify this section

Reply:
I'm trying to add Prettier to our repo to standardize formatting, PR coming soon.

Reply:
https://github.com/DapperCollectives/VESSEL/pull/99

Reply:
@KsenijaZivkovic  With above PR, all new changes will be auto-formatted during commit. But since this file is committed before that change, please rebase and run `npx prettier --write .` to get it formatted.